templates/style/_navbar.scss

```diff
@@ -349,6 +349,14 @@ div.nav-container {
         li a {
             overflow-x: hidden;
             text-overflow: ellipsis;
+
+            &.current {
+                font-weight: bold;
+                display: list-item;
+                list-style-type: disc;
+                list-style-position: inside;
+                padding-left: 4px;
```

Comment:
I needed to reduce the padding a bit to ensure the current platform is still aligned with the others (because of the bullet).

Reply:
Given the bullet is decided by the browser, I assume this distance is actually browser dependent?

As an alternative maybe something like

```css
.current::before {
  content: "•";
  position: absolute;
  margin-left: -10px;
}
.current {
  font-weight: bold;
}
```

That way the element itself is always aligned, and any weirdness about the bullet size will just cause it to be mispositioned.

Reply:
I was wondering which approach would be the best because `content` depends on the font but I suppose we have a better control over the font rather than the web browser so your solution sounds to be the best. Thanks!